src/yapf.cfg

```diff
@@ -0,0 +1,6 @@
+# Style settings example shown in
+# https://github.com/google/yapf/blob/master/README.rst
+[style]
+based_on_style = pep8
+spaces_before_comment = 4
+split_before_logical_operator = true
```

Comment:
@TotoGaz - these are settings used to format the files in the PR.  There are many more options compared to black if we want to tinker: https://github.com/google/yapf#knobs

Reply:
Similar to uncrustify, the user can format/check the python files by running `make yapf_style` or `make_yapf_check`